docker-storage-setup.sh

```diff
@@ -35,6 +35,39 @@ STORAGE_IN_FILE="/etc/sysconfig/docker-storage-setup"
 STORAGE_OUT_FILE="/etc/sysconfig/docker-storage"
 STORAGE_DRIVERS="devicemapper overlay overlay2"
 
+usage() {
```

Comment:
Shouldn't all this code go after comment `Main Script`. I mean that's where most the executable code starts. Things above that are supposed to be helper functions and variables.